sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -410,8 +410,13 @@ export class BatchingReceiverLite {
             // a chance to have fewer messages internally that could get lost if the user's
             // app crashes in receiveAndDelete mode.
             if (totalWaitTimer) clearTimeout(totalWaitTimer);
-
-            totalWaitTimer = setTimeout(actionAfterWaitTimeout, getRemainingWaitTimeInMs());
+            const remainingWaitTimeInMs = getRemainingWaitTimeInMs();
+            totalWaitTimer = setTimeout(() => {
+              logger.verbose(
+                `${loggingPrefix} Batching, waited for ${remainingWaitTimeInMs} milliseconds after receiving the first message.`
+              );
+              return finalAction();
```

Comment:
Nit: We dont need the `return` keyword here right? It returns void

Reply:
I only moved it around, not needed. Removed